### ISSUE TITLE
podman build with remote is always silent (logging level does not work)

### ISSUE THREAD

**Issue by ssbarnea** [containers/podman#7498]
**Description**

podman build seems totally silent. when run locally this does not happen.

Apparently the entire stdout/stderr is lost.

**Steps to reproduce the issue:**

```
export CONTAINER_HOST=ssh://root@somehost/run/podman/podman.sock
podman build
```

**Describe the results you received:**

no stdout/stderr whatsover/

**Describe the results you expected:**

Some stderr logging information, especially when using `--loglevel 1,2,3`

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
(paste your output here)
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.15.1
  cgroupVersion: v2
  conmon:
    package: conmon-2.0.19-1.fc32.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.19, commit: 5dce9767526ed27f177a8fa3f281889ad509fea7'
  cpus: 8
  distribution:
    distribution: fedora
    version: "32"
  eventLogger: file
  hostname: leno
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 5.6.8-300.fc32.x86_64
  linkmode: dynamic
  memFree: 8360042496
  memTotal: 33059672064
  ociRuntime:
    name: crun
    package: crun-0.14.1-3.fc32.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.14.1
      commit: 598ea5e192ca12d4f6378217d3ab1415efeddefa
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  rootless: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 16601051136
  swapTotal: 16601051136
  uptime: 2815h 5m 12.95s (Approximately 117.29 days)
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Thu Jan  1 01:00:00 1970
  GitCommit: ""
  GoVersion: go1.14.6
  OsArch: linux/amd64
  Version: 2.0.5
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by mheon** [comment]
I believe @jwhonce is working on this in https://github.com/containers/podman/pull/7452

**Comment by afbjorklund** [comment]
It doesn't seem to be "just" the logging, I don't actually get any image built either...

```console
 podman1-remote build .
STEP 1: FROM busybox
Getting image source signatures
Copying blob sha256:61c5ed1cbdf8e801f3b73d906c61261ad916b2532d6756e7c4fbcacb975299fb
Copying config sha256:018c9d7b792b4be80095d957533667279843acf9a46c973067c8d1dff31ea8b4
Writing manifest to image destination
Storing signatures
STEP 2: RUN true
STEP 3: COMMIT
--> 7e7fa3f44ea
7e7fa3f44eac8d87b87fb232c3b40a07b08f1661ef8bd86e4debd3a59086280a
 podman1-remote images
REPOSITORY                  TAG      IMAGE ID       CREATED              SIZE
<none>                      <none>   7e7fa3f44eac   About a minute ago   1.45 MB
docker.io/library/busybox   latest   018c9d7b792b   4 weeks ago          1.45 MB

```

```console
 podman2-remote build .
 podman2-remote images
REPOSITORY                 TAG     IMAGE ID      CREATED      SIZE
docker.io/library/busybox  latest  018c9d7b792b  4 weeks ago  1.45 MB

```

The base image is there, because I ran a simple "run" test before the "build".

**Comment by afbjorklund** [comment]
Probably related to #7136 and #7137

**Comment by ashley-cui** [comment]
closed via #7452
